init.lua

```diff
@@ -189,4 +188,23 @@ torch.Tensor.isTensor = torch.isTensor
 -- remove this line to disable automatic heap-tracking for garbage collection
 torch.setheaptracking(true)
 
+function torch.multinomialAliasSetup(probs, state)
+    local prob_tbl, alias_tbl
+    if state then 
+      assert(torch.type(state) == 'table')
+      state[1], state[2] = torch.multinomialAliasSetup_(probs, state[1], state[2])
+    else 
+      prob_tbl, alias_tbl = torch.multinomialAliasSetup_(probs)
+      state = {prob_tbl, alias_tbl}
+    end
+    return state
+end
+
+function torch.multinomialAlias(output, state)
+   local prob_tbl, alias_tbl
```

Comment:
This one too, right ? I couldn't figure a war around it. Unlike `torch.multinomial` this one doesn't take a `FloatTensor` as the first argument but `output` which  is a `DoubleTensor`.